src/DateField/DateField.tsx

```diff
@@ -86,31 +55,26 @@ export class DateField extends React.Component<Props, State> {
 		}
 	};
 
-	public handleOpen = () => this.setState({ isOpenCalendar: true });
-
-	public handlePrevMonth = () => {
-		const oldDate = this.state.showedDate;
-		oldDate.setMonth(oldDate.getMonth() - 1);
-		this.setState({ showedDate: oldDate });
-	};
-
-	public handleNextMonth = () => {
-		const oldDate = this.state.showedDate;
-		oldDate.setMonth(oldDate.getMonth() + 1);
-		this.setState({ showedDate: oldDate });
+	public handleOpen = () => {
+		if (!this.props.disabled) {
+			this.setState({ isOpenCalendar: true })
+		}
 	};
 
-	public handleChangeYear = (event: any) => {
-		const oldDate = this.state.showedDate;
-		const value = parseInt(event.target.value, 10);
-		oldDate.setFullYear(value);
+	public handleChangeCurrentMonth = (direction: 1 | -1) => () => {
```

Comment:
direction лучше всё же сделать енумом, что бы было более понятно
```ts
enum Direction {
  Prev = -1,
  Next = 1
}
```
```suggestion
	public handleChangeCurrentMonth = (direction: Direction) => () => {
```